OpenEdXMobile/src/main/java/org/edx/mobile/view/CourseAnnouncementsActivity.java

```diff
@@ -43,10 +44,12 @@ protected void onCreate(Bundle savedInstanceState) {
                 .getSerializable(Router.EXTRA_COURSE_DATA);
 
         //check courseData again, it may be fetched from local cache
-        if ( courseData != null ) {
+        if (courseData != null) {
             activityTitle = courseData.getCourse().getName();
-
-            environment.getSegment().trackScreenView(courseData.getCourse().getName());
+            environment.getSegment().trackScreenView(
+                    ISegment.Screens.COURSE_ANNOUNCEMENTS,
+                    courseData.getCourse().getId(),
+                    null);
```

Comment:
Will be considering in https://openedx.atlassian.net/browse/MA-2978. We might end up using it, or might not.